opm/simulators/linalg/gpuistl/set_device_indirection.cpp

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
I am not a fan of this name. First this file includes two functions, one of which has nothing to do with setting the device, secondly, what indirection refers to is a bit unclear (and for a user of said file, maybe not needed).

May I suggest renaming to `device_management.hpp` ? and then include this file in the main file.

The file now named `set_device.hpp` should probably be moved to `detail` (and renamed to something like `device_management`), as you don't want this file to be included from outside `gpuistl` (and the `detail` namespace signals this)

Reply:
done